cypress/utility.ts

```diff
@@ -0,0 +1,2 @@
+export const dataSynth = (selector: string) => `[data-synth="${selector}"]`;
+export const partialDataSynth = (selector: string) => `[data-synth*=${selector}]`;
```

Comment:
Good approach, but the locator is still hard coded from respective functions. Can we have common place for storing locators ?

Reply:
I am following POM so as far as I know all the locators I have placed in respective pages. Please clarify it if you meant something else entirely.